app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -282,6 +289,16 @@ public boolean onItemLongClick(AdapterView<?> parent, View v, int pos, long id)
         applyDesignTweaks();
     }
 
+    private void adjustInputType(String currentText) {
+        if (currentText != null && Pattern.matches("[+]\\d+", currentText)) {
+            searchEditText.setInputType(InputType.TYPE_CLASS_PHONE);
+        } else if (searchEditTextSpellcheck) {
+            searchEditText.setInputType(SPELLCHECK_ENABLED_TYPE);
```

Comment:
Ideally, this should be `InputType.TYPE_CLASS_TEXT |
            InputType.TYPE_TEXT_FLAG_AUTO_CORRECT;`

You can reuse the static value `SPELLCHECK_ENABLED_TYPE`.